PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,23 @@
-Please try to adhere to the guidelines below as much as possible when submitting your pull request.
+<!--
+Thanks for contributing a pull request! Please ensure you have taken a look at
+the [contribution guidelines](https://github.com/scikit-learn/scikit-learn/blob/master/CONTRIBUTING.md))
```

Comment:
I wouldn't use markdown here either as mentioned above.

It'd be great if there was a section in CONTRIBUTING.md you could link to, which was specific to guidelines related to creating a pull request.